src/wasm-lib/kcl/src/parser/snapshots/kcl_lib__parser__parser_impl__snapshot_tests__bi.snap

```diff
@@ -0,0 +1,271 @@
+---
+source: kcl/src/parser/parser_impl.rs
+expression: actual
+---
+{
+  "start": 0,
+  "end": 100,
+  "body": [
+    {
+      "type": "VariableDeclaration",
+      "type": "VariableDeclaration",
```

Comment:
Missed one?

Reply:
So, I added the test with ID `bi` while debugging our problem. Then when I fixed the problem, I deleted the unnecessary test, and forgot to remove this file too :)

So it's actually fine! I will delete this once CI runs to avoid retriggering it.